src/charm.py

```diff
@@ -141,6 +149,26 @@ def __init__(self, *args):
 
         self.log_forwarder = LogForwarder(self)
 
+        self.loki_device_alert_rules_path = "./loki_alert_rules"
+        self.loki_device_alerts_push_api_consumer = LokiPushApiConsumer(
+            charm=self,
+            relation_name="logging-devices-alerts",
+            alert_rules_path=self.loki_device_alert_rules_path,
+            # The alerts we are sending are not specific to
+            # cos-registration-server but to devices outside of juju
+            skip_alert_topology_labeling=True,
+        )
+
+        self.prometheus_device_alert_rules_path = "./prometheus_alert_rules"
+        self.prometheus_device_alerts_remote_write_consumer = PrometheusRemoteWriteConsumer(
+            charm=self,
+            relation_name="send-remote-write-devices-alerts",
```

Comment:
Why such a convoluted name?

Reply:
Default name is: `send-remote-write` so I added `devices` and `alerts` to avoid confusion that this is sending devices metrics.